server/package.json

```diff
@@ -33,19 +33,21 @@
         "@nestjs/serve-static": "^2.2.2",
         "@nestjs/typeorm": "^7.1.5",
         "bcrypt": "^5.0.1",
+        "class-transformer": "^0.4.0",
```

Comment:
i don't see this one being used anywhere
let's remove until it is being used

Reply:
actually, we may need it 😕 
just tried to start BE server without it and got errors 🤷

Reply:
Reinstalled it.